scm/src/scm_type_defs.F90

```diff
@@ -1064,7 +1064,7 @@ subroutine physics_set(physics, scm_input, scm_state)
       !
       ! Orographical data (2D)
       !
-      if (scm_state%model_ics) then
+      if (scm_state%model_ics .or. scm_state%lsm_ics) then
```

Comment:
@dustinswales I don't understand why this is needed. If one is using only ICs for LSMs (i.e. the GABLS3 canned cases), you don't need all of the orographic data. Are you saying that we need `landfrac`, `lakefrac` and `lakedepth` as additional variables when importing LSM ICs?

Reply:
Yeah, if the LSM is coupled to a lake model, we need landfrac, lakefrac and lakedepth as ICs.
I didn't want to add a new flag for "lake_ics", so I did this. I'm open to alternatives.

Reply:
It's weird that these variables are included in the "oro" files rather than the "sfc" files. I'm going to push a commit to this that needs to be tested for both a canned GABLS3 case and a UFS lake case to make sure that it's doing the right thing. As-is, this will break the GABLS3 canned cases, I think.

Reply:
I'm going to test it real quick before I try to "fix" anything.

Reply:
@dustinswales Please see https://github.com/dustinswales/ccpp-scm/pull/13.